website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -1246,35 +1246,3 @@ spec:
 status:
   instanceProfile: "${CLUSTER_NAME}-0123456778901234567789"
 ```
-## status.conditions
```

Comment:
Shouldn't this be included?

Reply:
Good catch, looks like this never got added to the preview docs, only the v0.36 docs. Updated.